scenario-runner/my_scenario.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2023 Leidos
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import py_trees
+
+from srunner.scenariomanager.carla_data_provider import CarlaDataProvider
```

Comment:
Can we include a requirements.txt for pip to keep track of the dependent packages to run this script

Reply:
`srunner` is not maintained on [PyPi](https://pypi.org/). There is a package hosted there, but it's from 2019. I will add `srunner` to a `requirements.txt` file, but it will need to be commented out with an explanation.